app/graphql/albums/resolver.js

```diff
@@ -0,0 +1,11 @@
+const { getAlbum, getAlbumPhotos } = require('../../services/album');
+
+exports.dataAlbum = async (_, params) => {
+  const albums = await getAlbum(params);
+  const albumsPhotos = await getAlbumPhotos(params.id);
+  return {
+    ...albums,
+    artist: albums.userId,
```

Comment:
be carefull, you are not replacing userId by artist. You are just adding a new property artist. When you use the spread operator for album, you add `userId` to the response